wavefront_pyformance/wavefront_reporter.py

```diff
@@ -112,15 +112,15 @@ class WavefrontProxyReporter(WavefrontReporter):
     def __init__(self, host, port=2878, distribution_port=None,
                  source='wavefront-pyformance', registry=None,
                  reporting_interval=10, clock=None, prefix='proxy.',
-                 tags=None):
+                 tags=None, tracing_port=None):
         """Run parent __init__ and do proxy reporter specific setup."""
         super(WavefrontProxyReporter, self).__init__(
             source=source, registry=registry,
             reporting_interval=reporting_interval, clock=clock, prefix=prefix,
             tags=tags)
         self.wavefront_client = wavefront_sdk.WavefrontProxyClient(
```

Comment:
The reason we don't expose tracing port is that `pyformance` is basically a metrics library, which would not be reporting tracing data. But you can still create the wavefront client yourself and pass it as `wavefront_client` manually.
Any use case you want to have the tracing port directly here?

Reply:
Yes, I'm currently working on a HOL on polyglot application (beachshirts). Due to the nature of HOL having a restricted environment, I had to use proxy mode to use http proxy, which would make the work a whole lot easier. Then, I found out that the application was not running, and found out the warehouse service (which was made and run using python) was not able to receive the tracing_port from the reporter. I had to add the tracing_port as an init input to finally make it work.